File: GoEnv/go/common/EdocDistCert/Test/BlabEdocDistGetCertClientTest.go
package main

import (
	blabClient "../Client"
	testClient "./TestConfig"
	"log"
)

// 전자문서 유통증명서 발급 테스트
func main() {
	log.SetFlags(log.LstdFlags | log.Lshortfile)
	err := testClient.BlabTestClientInit()
	if err != nil {
		return
	}

	clientConfig := testClient.ClientConfig
	appRes, err := blabClient.PostEdocDistGetCert(clientConfig.Server.BaseUrl, clientConfig.Individual.EdocNum, clientConfig.Individual.Eaddr, clientConfig.Individual.Reason)
	if err != nil {
		log.Printf("err=%+v\n", err)
		return
	}

	log.Printf("appRes=%+v", appRes)
	log.Printf("appRes.Data=%+v", appRes.Data)
}

File: GoEnv/go/common/EdocDistCert/Test/node/BlabEdocDistGetCertClientTest.mjs
import blabClient from './BlabClient.mjs';
import clientConfig from './BlabClientConfig.mjs';

// 미작업: 전자문서 유통증명서 발급 테스트
function BlabEdocDistGetCertTest() {
    blabClient.PostEdocDistGetCert(
        clientConfig.server.baseUrl,
        clientConfig.individual.edocNum,
        clientConfig.individual.eaddr,
        clientConfig.individual.reason)
    .then(res => {
        // console.log(res);
    });
}

BlabEdocDistGetCertTest();